cmd/argo/commands/server.go

```diff
@@ -21,13 +21,14 @@ import (
 
 func NewServerCommand() *cobra.Command {
 	var (
-		logLevel         string // --loglevel
-		authMode         string
-		configMap        string
-		port             int
-		baseHRef         string
-		namespaced       bool   // --namespaced
-		managedNamespace string // --managed-namespace
+		logLevel           string // --loglevel
+		authMode           string
+		configMap          string
+		port               int
+		baseHRef           string
+		namespaced         bool   // --namespaced
+		managedNamespace   string // --managed-namespace
+		disableOpenBrowser bool
```

Comment:
I think this really should not be the default, i.e. should be `enableOpenBrowser`

Reply:
@jessesuen can you please arbitrate on this? @sarabala1979 thinks the browser should open by default, I think not. 

My take is that this might be problematic for community users.  E.g. running on other architectures, or using different configurations. I also think that it might be unreliable/flakey.

I'll let @sarabala1979 give his rational.